File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        length = len(nums)
        res = [1 for _ in range(length)]
        pre = 1
        for i in range(length):
            res[i] = pre
            pre *= nums[i] 
        post = 1
        for i in range(length-1,-1,-1):
            res[i]*=post
            post*=nums[i]
        return res


        

File: Data Structures & Algorithms/remove-element/submission-0.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
        
        int curr = 0;
        for(int i=0; i<nums.Length;i++)
        {
            if(nums[i] == val)
            {
                continue;
            }
            nums[curr] = nums[i];
            curr += 1;
        }
        return curr;
    }
}